src/bin/genesis.rs

```diff
@@ -0,0 +1,67 @@
+use dotenv::dotenv;
+use ethers::types::U256;
+use kakarot_rpc::test_utils::katana::genesis::KatanaGenesisBuilder;
+use lazy_static::lazy_static;
+use reth_primitives::B256;
+use serde::Serialize;
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet_crypto::FieldElement;
+use std::{
+    collections::HashMap,
+    env::var,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+
+lazy_static! {
+    static ref GENESIS_FOLDER_PATH: PathBuf = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join(".katana");
+    static ref KAKAROT_CONTRACTS_PATH: PathBuf =
+        Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf().join("lib/kakarot/build");
+    static ref COINBASE_ADDRESS: FieldElement = FieldElement::from(0x12345u32);
+    static ref SALT: FieldElement = FieldElement::ZERO;
+}
+
+#[serde_as]
+#[derive(Serialize)]
+struct Hex(#[serde_as(as = "UfeHex")] pub FieldElement);
+
+#[derive(Serialize)]
+struct KatanaManifest {
+    declarations: HashMap<String, Hex>,
+    deployments: HashMap<String, Hex>,
+}
+
+fn main() {
+    // Load the env vars.
+    dotenv().ok();
+
+    // Read the env vars.
+    let pk = B256::from_str(&var("EVM_PRIVATE_KEY").expect("Missing EVM private key"))
+        .expect("Failed to parse EVM private key");
+
+    // Read all the classes.
+    let mut builder = KatanaGenesisBuilder::default()
+        .load_classes(KAKAROT_CONTRACTS_PATH.clone())
+        .with_kakarot(*COINBASE_ADDRESS)
+        .expect("Failed to set up Kakarot");
```

Comment:
is it typical in builder patterns to have error paths?
should we use expect on those?

Reply:
I mean sure I think builders can error, not sure why you would not error out. Here is a discussion for the builder pattern in Java: https://stackoverflow.com/questions/15590325/builder-pattern-with-error-checking-is-it-possible-advisable. Also these lines are from a binary, so I can't do anything else then expect.

Reply:
I think eyre helps in that regards for custom errors but expect seems ok!